### ISSUE TITLE
Firefox error: addon appears corrupt

### ISSUE THREAD

**Issue by xanda-escuyer** [puemos/hls-downloader#327]
**Describe the bug**
The addon will not install to FireFox. Applies to  package archive from GitHub and built from source.

**Screenshots**
![Screenshot from 2024-04-07 17-31-15](https://github.com/puemos/hls-downloader/assets/15840995/578bae5a-fddd-4858-ac51-2a38cfe0097a)

**Desktop (please complete the following information):**
 - OS: Debian 12 Bookworm/Linux 6.1.0-17-amd64
 - Browser: Firefox
 - Version: 115.9.1esr

**Additional context**
the build instructions might also need changing as the command `sh ./build.sh` fails with:-
```
$: ./build.sh 
sh: 0: cannot open ./scripts/clean.sh: No such file
sh: 0: cannot open ./scripts/build-core.sh: No such file
sh: 0: cannot open ./scripts/build-design-system.sh: No such file
sh: 0: cannot open ./scripts/build-extension.sh: No such file
	zip warning: name not matched: ./dist/.

zip error: Nothing to do! (try: zip -r extension-archive.zip . -i ./dist/.)

```
This was corrected instead by running from the top directory: `sh scripts/build.sh

`Build log attached:
[build.log](https://github.com/puemos/hls-downloader/files/14898086/build.log)

**Comment by puemos** [comment]
FYI crx is now available in [v4.0.1](https://github.com/puemos/hls-downloader/releases/tag/v4.0.1)

**Comment by xanda-escuyer** [comment]
![Screenshot from 2024-04-10 19-40-50](https://github.com/puemos/hls-downloader/assets/15840995/1b877cc3-da5c-47e6-a8cb-64aff645cd21)

This applies to both the release package and when built from source.
